packages/feedback-service/src/resolver.ts

```diff
@@ -1,70 +1,32 @@
+import * as _ from "lodash";
 import { Feedback } from './schema';
-import { getTransporter } from './helpers';
-  
-const transporter = getTransporter;
 
 export const FeedbackResolver = {
   Query: {
-    //queries
     listFeedback(root: any, args: any, ctx: any) {
       return Feedback.find()
         .sort({ "timestamp": -1 })
         .then(response => response)
         .catch(err => err);
     },
     getFeedback(root: any, args: any, ctx: any) {
-      return Feedback.findById(args._id)
+      return Feedback.findById(args.id)
         .then(response => response)
         .catch(err => err);
+    },
+    getFeedbackBy(root: any,  args : any, ctx: any){
+      const searchBy = _.pick(args, ["input"]);
+      return Feedback.find(searchBy.input)
+          .then(response=>response)
+          .catch(err=>err);
```

Comment:
```suggestion
      return Feedback.find(searchBy.input).exec();
```

Reply:
if yo can you help me in understand why .then .catch are useless are i seen other developer also implement the same.

Reply:
The `.then` and `.catch` are used to handle certain cases. For eg. then is used when you want to perform certain operations on the data, and catch is used when you want to handle the error in some other way.
Here, you were just return the arguments as it is, so it did not have any effect.
Instead, graphql expects the resolvers to return a promise, and it can take care of the rest, so just `.find().exec()` is sufficient.

Reply:
Got it.